pkg/cmd/experimental/syncgroups/ad/augmented_ldapinterface_test.go

```diff
@@ -0,0 +1,121 @@
+package ad
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/go-ldap/ldap"
+	"github.com/openshift/origin/pkg/auth/ldaputil"
+	"github.com/openshift/origin/pkg/auth/ldaputil/ldapclient"
+	"github.com/openshift/origin/pkg/auth/ldaputil/testclient"
+	"github.com/openshift/origin/pkg/cmd/experimental/syncgroups/interfaces"
+)
+
+var _ interfaces.LDAPMemberExtractor = &AugmentedADLDAPInterface{}
```

Comment:
Congrats on finding a spot where Jordan and I are opposed.  I like these where the types are defined to help people see it. He'll come around eventually.  Put these back in the main source